terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/test/java/org/terasoluna/gfw/common/codelist/i18n/SimpleI18nCodeListTest.java

```diff
@@ -21,29 +21,49 @@
 import java.util.Locale;
 import java.util.Map;
 
+import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Lazy;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration(locations = {
         "classpath:org/terasoluna/gfw/common/codelist/i18n/simpleI18nCodeList.xml" })
+@Lazy
```

Comment:
- [ ] Please comment why using `@Lazy`. (expect fixed locale when `Locale.getDefault()`).

Reply:
Modified by 273f0ef
Added javadoc and adjusted the position of `@Lazy`

Reply:
It is strange to write a class level Javadoc only to explain why `@Lazy` is granted.
It seems be enough to write on source code comment.

Reply:
By the way, is it possible to avoid `@Lazy` by setting the locale in the constructor?

Reply:
There was no problem with the use of the constructor.
dc44f0e8587b57911d6b630e837a0c36a6a34a26